CalendarFXView/src/main/java/com/calendarfx/model/Entry.java

```diff
@@ -21,9 +21,7 @@
 
 import java.text.ParseException;
 import java.time.*;
-import java.util.HashMap;
-import java.util.Objects;
-import java.util.Optional;
+import java.util.*;
```

Comment:
I do not use wildcard imports. Can you change this in your IDE settings please? Otherwise we keep getting non-relevant deltas in our files.